docs/compilers/CSharp/Runtime Async Design.md

```diff
@@ -255,38 +284,22 @@ Translated C#:
 
 ```cs
 var local = C.M();
-var i =
-{
-    var awaiter = local.GetAwaiter();
-    if (!awaiter.IsComplete)
-    {
-        /* Runtime-Async Call */ System.Runtime.CompilerServices.RuntimeHelpers.AwaitAwaiterFromRuntimeAsync<System.Runtime.CompilerServices.TaskAwaiter>(awaiter);
-    }
-    awaiter.GetResult()
-};
+var i = System.Runtime.CompilerServices.RuntimeHelpers.Await<int>(local);
```

Comment:
The paragraph above still mentions `AwaitAwaiterFromRuntimeAsync` and `UnsafeAwaitAwaiterFromRuntimeAsync`. Can you clarify when we use those vs. `Await`/`Await<T>`?